jetbrains-core/src/software/aws/toolkits/jetbrains/core/DeleteResourceAction.kt

```diff
@@ -34,11 +35,18 @@ abstract class DeleteResourceAction<in T : AwsExplorerResourceNode<*>>(text: Str
 
             if (response != null) {
                 ApplicationManager.getApplication().executeOnPooledThread {
+                    val future = CompletableFuture<T>()
                     try {
                         performDelete(selected)
-                        notifyInfo(message("delete_resource.deleted", selected.resourceType, resourceName))
+                        future.complete(selected)
```

Comment:
Why use the future, if you are always on the same thread anyway?

Reply:
definitely don't see the point of the future here

Reply:
Reverted to not use Future.